backend/config.go

```diff
@@ -48,6 +48,7 @@ type AppConfig struct {
 	ShowTrackChangeNotification bool
 	EnableLrcLib                bool
 	SkipSSLVerify               bool
+	QueueSize		    int
```

Comment:
Let's rename this to "EnqueueBatchSize", and this same setting can then be re-used for the "continuous queuing" feature I plan to add